test/sumo_basic_SUITE.erl

```diff
@@ -70,6 +71,11 @@ delete_all(_Config) ->
 delete(_Config) ->
   run_all_stores(fun delete_module/1).
 
+check_datetime_fields(_Config) ->
+  lists:foreach(
+    fun do_check_datetime_fields/1,
+    [sumo_test_people_riak]).
```

Comment:
We successfully managed to remove particular stores from the test suites. You should not have `sumo_test_people_riak` here. If you really need this and if this can only be tested in the riak store, then the list `[sumo_test_people_riak]` should come from `sumo_test_utils:people_with_datetime()` or something similar.
But I wonder why would sumo_test_people_riak have a function called `datetime` and other modules would not.
From my perspective, the list should be `sumo_test_utils:all_people()` and you should implement the `datetime/1` function in all sumo_test_people's implementations.